_posts/en/newsletters/2025-11-21-newsletter.md

```diff
@@ -0,0 +1,261 @@
+---
+title: 'Bitcoin Optech Newsletter #381'
+permalink: /en/newsletters/2025/11/21/
+name: 2025-11-21-newsletter
+slug: 2025-11-21-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter looks at an analysis of how block propagation times may
+affect miner revenue and describes a new approach for resolving protocols where
+multiple parties share funds. Also included are our regular sections describing
+recent changes to services and client software and summarizing recent merges to
+popular Bitcoin infrastructure software.
+
+## News
+
+- **Modeling stale rates by propagation delay and mining centralization:**
+  Antoine Poinsot [posted to Delving Bitcoin][antoine delving] about modeling
+  stale block rates and how block propagation time affects a miner's revenue as
+  a function of its hashrate given propagation time is zero. He set up a
```

Comment:
The sentence here doesn't make sense

> how block propagation time affects a miner’s revenue as a function of its hashrate given propagation time is zero

Reply:
My apologies for missing it in the final review